cmd/launcher/interactive.go

```diff
@@ -19,22 +21,17 @@ import (
 )
 
 func runInteractive(systemMultiSlogger *multislogger.MultiSlogger, args []string) error {
-	flagset := flag.NewFlagSet("interactive", flag.ExitOnError)
-	var (
-		flDebug        = flagset.Bool("debug", false, "enable debug logging")
-		flOsquerydPath = flagset.String("osqueryd_path", "", "The path to the oqueryd binary")
-		flOsqueryFlags launcher.ArrayFlags
-	)
-
-	flagset.Var(&flOsqueryFlags, "osquery_flag", "Flags to pass to osquery (possibly overriding Launcher defaults)")
-
-	flagset.Usage = commandUsage(flagset, "launcher interactive")
-	if err := flagset.Parse(args); err != nil {
+	opts, err := launcher.ParseOptions("interactive", args)
+	if err != nil {
 		return err
 	}
 
+	if opts.RootDirectory == "" {
+		opts.RootDirectory = launcher.DefaultPath(launcher.RootDirectory)
```

Comment:
Curious what this is used for since we also have the temp root dir?

Reply:
the opts.RootDirectory is where to look for the kv.sqlite db to load the ATC config from, the temp root dir is for interactive to store temp files like socket and aegues lense files.

I added some comments to clarify the 2